evap/evaluation/migrations/0158_unified_questions_from_tmp_relation.py

```diff
@@ -0,0 +1,127 @@
+# Generated by Django 5.2 on 2025-10-27 17:17
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def question_assignments_to_questions(apps, _schema_editor):
+    Question = apps.get_model("evaluation", "Question")
+    QuestionAssignment = apps.get_model("evaluation", "QuestionAssignment")
+    RatingAnswerCounter = apps.get_model("evaluation", "RatingAnswerCounter")
+    TextAnswer = apps.get_model("evaluation", "TextAnswer")
+    questions = {}
+    for question_assignment in QuestionAssignment.objects.all().prefetch_related("question"):
+        new_question = question_assignment.question
+        questions[question_assignment.pk] = Question(
+            text_de=new_question.text_de,
+            text_en=new_question.text_en,
+            allows_additional_textanswers=new_question.allows_additional_textanswers,
+            type=new_question.type,
+            questionnaire=question_assignment.questionnaire,
+            order=question_assignment.order,
+        )
+    Question.objects.bulk_create(questions.values())
+
+    def set_question(answer):
+        answer.question_id = questions[answer.question_assignment_id].pk
+        return answer
+
+    for model in (RatingAnswerCounter, TextAnswer):
+        model.objects.bulk_update(
+            map(set_question, model.objects.all()),
+            fields=["question"],
+        )
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("evaluation", "0157_temporary_newquestion_relation"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=migrations.RunPython.noop,
+            reverse_code=question_assignments_to_questions,
+        ),
+        migrations.AlterUniqueTogether(
+            name="ratinganswercounter",
+            unique_together=set(),
+        ),
+        migrations.RemoveField(
+            model_name="ratinganswercounter",
+            name="question",
+        ),
+        migrations.RemoveField(
+            model_name="textanswer",
+            name="question",
+        ),
+        migrations.DeleteModel(
+            name="Question",
+        ),
+        migrations.RenameField(
+            model_name="textanswer",
+            old_name="question_assignment",
+            new_name="question",
+        ),
```

Comment:
These are not needed because we rename them again later, right? We could also name them "assignment" in migration 157 already so that we don't have to rename at all after that